platform-sdk/swirlds-platform-core/src/main/java/com/swirlds/platform/reconnect/emergency/EmergencyReconnectTeacher.java

```diff
@@ -125,7 +127,18 @@ public void execute(final Connection connection) {
                     writeHasState(connection, false);
                     logger.info(
                             RECONNECT.getMarker(),
-                            "Peer {} requested to perform an emergency reconnect but no compatible state was found.",
+                            "Peer {} requested state ({},{}) to perform an emergency reconnect but no compatible state was found."
+                                    + "The emergency state we have is ({},{})",
+                            round,
```

Comment:
These are out of order - `otherId` belongs in the first slot

Reply:
good catch, fixed